__mocks__/durable-functions.ts

```diff
@@ -0,0 +1,9 @@
+export const mockStartNew = jest.fn();
```

Comment:
we won't use durable, we can remove the module

Reply:
Fix with 066daa4